lib/core/src/lib/info-drawer/info-drawer.component.ts

```diff
@@ -79,4 +88,45 @@ export class InfoDrawerComponent {
     onTabChange(event: MatTabChangeEvent) {
         this.currentTab.emit(event.index);
     }
+
+    getInfoDrawerIcon(node: Node) {
```

Comment:
Please add the returns type to all methods here.

Reply:
Added